libraries/azurerm_sql_servers.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+require 'json'
+
+class AzurermSqlServers < AzurermPluralResource
+  name 'azurerm_sql_servers'
+  desc 'Verifies settings for a collection of Azure SQL Servers'
+  example <<-EXAMPLE
+    describe azurerm_sql_servers do
+        it  { should exist }
+    end
+  EXAMPLE
+
+  attr_reader :table
+
+  FilterTable.create
+             .register_column(:ids,        field: :id)
+             .register_column(:names,      field: :name)
+             .register_column(:kinds,      field: :kind)
+             .register_column(:locations,  field: :location)
+             .register_column(:properties, field: :properties)
+             .register_column(:tags,       field: :tags)
```

Comment:
Should this be:
```
              .register_column(:tags,       field: :tag) 
```

Reply:
`tags` is the field name in the singular resource https://github.com/inspec/inspec-azure/blob/198a837941e094e79d9860bab183cc8c73efd408/libraries/azurerm_sql_server.rb#L20 similar to `properties`. Should it still be `tag`?

Reply:
The use-case I was thinking of is filtering on a single tag e.g. 
```
  describe azurerm_sql_servers(resource_group).where(tag: "my_very_special_tag") do
    it { should exist }
    # you wouldn't need to do the below here but for illustration purposes
    its('tags') { should include "my_very_special_tag" } 
  end
```
The singular resource properties aren't affected, could be named the same or differently.  Might be worth checking if `properties` works as expected because I've only ever seen the plural/singular case.